props/saber_fett263_buttons.h

```diff
@@ -845,35 +5178,89 @@ SaberFett263Buttons() : PropBase() {}
           beeper.Beep(0.5, 3000);
         }
         return;
+#ifndef FETT263_SAY_BATTERY
       case EFFECT_BATTERY_LEVEL:
         if (SFX_battery) {
           hybrid_font.PlayCommon(&SFX_battery);
         } else {
           beeper.Beep(0.5, 3000);
         }
         return;
+#endif
       case EFFECT_FAST_ON:
         if (SFX_faston) {
           hybrid_font.PlayCommon(&SFX_faston);
         }
         return;
-      default: break; // avoid compiler warnings
+      default:
+        break; // avoid compiler warnings
     }
   }
 
 private:
-  bool pointing_down_ = false;
   bool swing_blast_ = false;
-  bool mode_volume_ = false;
+  bool check_blast_ = false;
   bool auto_lockup_on_ = false;
   bool auto_melt_on_ = false;
   bool battle_mode_ = false;
-  uint32_t thrust_begin_millis_ = millis();
-  uint32_t push_begin_millis_ = millis();
-  uint32_t clash_impact_millis_ = millis();
-  uint32_t last_twist_ = millis();
-  uint32_t last_push_ = millis();
-  uint32_t saber_off_time_ = millis();
+  bool menu_ = false;
```

Comment:
Please add a comment for each new variable explaining what it's for.

Reply:
comments added